### PR TITLE
Refactorings

### PR DESCRIPTION
**Pull request by inyono**
See commits for the list of changes. Probably easiest to review them one by one 😬

### CODE REVIEW

File: src/styles/text.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { WHITE } from './colors';
+
+export const FONT_FAMILY = {
+  fontFamily: 'norddruck'
+};
+
+export const DEFAULT = {
+  ...FONT_FAMILY,
+  color: WHITE,
+  fontSize: 35,
```
Comment (Knorrke) [inline src/styles/text.js]: fontSize was 40 before, is this change intentional?

**Comment by Knorrke** [comment]
I have some problems understanding what the deleted files (in commits [all: remove redux](https://github.com/serlo-org/serlo-abc/pull/46/commits/f654fd8a8121d423b6cb5c1fe720124342cec1d0) and [styles: extract text styles, use norddruck font](https://github.com/serlo-org/serlo-abc/pull/46/commits/8d12c90ba74e4a2373a402639a02ec9d98d42f2c#diff-1f21a70d0bb5e91b76add137466c9c58)) were for. Why aren't they needed anymore?

File: src/styles/text.js (hunk 1 above)
Comment (inyono) [inline src/styles/text.js]: Yeah, since the new font looks different. But the change is arguable.

**Comment by inyono** [comment]
Well, they were responsible for the "real app" workflow. But we have to rewrite that anyway, so I removed it for now. If we need some stuff, git has a backup.

**Comment by Knorrke** [comment]
Ok then, thanks for refactoring!

**Comment by inyono** [comment]
Thanks for reviewing :)
